### DISCUSSION TITLE
Why do I cannot create my organization?

### DISCUSSION THREAD

**Discussion by iamthanhlong** [community/community#53612] [New to GitHub]
### Select Topic Area

Question

### Body

Hi!

I was trying to create my own new organization, but the "Next" button wasn't visible so I couldn't get to the next step.

Help me with this please!

Thanks
![image](https://user-images.githubusercontent.com/109861002/234044623-09b8b267-c000-4e65-aa98-38cea2d66353.png)

**Comment by NJacksonDev** [comment]
It works for me - please make sure that you verify your account (have to select a picture/captcha in a window under "Verify your account" that appears). If that window/popup does not appear, try another browser or check current browser settings re: popups, etc.

**Reply by iamthanhlong** [reply]
Thank you so much for your instruction <3
